### ISSUE TITLE
showDndOverlay is not called always

### ISSUE THREAD

**Issue by mathuo** [mathuo/dockview#395]
### Discussed in https://github.com/mathuo/dockview/discussions/391

<div type='discussions-op-text'>

<sup>Originally posted by **garyvh2** November 15, 2023</sup>
So I needed to handle an specific scenario on which drag and drop is only enabled on the tab elements, but not on the rest of the targets. I wanted to use the `showDndOverlay` but seems like it's only executed for elements that come from the outside of the layout.

Is there any alternative that I might try?

TL;DR; I want to enable drag and drop only on tab elements (reorder, or drop on `void-container`)</div>

**Comment by mathuo** [comment]
Released in 1.10.0 https://dockview.dev/
